src/reistijden_v1/migrations/0012_remove_old_models.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.14 on 2022-08-01 15:03
+
```

Comment:
The automatic generated migrations seem to be slightly redundant, dropping fields of models that are deleted anyway.
After discussing this with Daniel, we dont expect a significant performance impact, and since this was tested, we are going with this migration,